README.md

```diff
@@ -53,8 +53,9 @@ then simply do:
   as the first part of the name of the staging environments created by you:
   `${BIOMAGE_NICK:-${USER}}-...`.
 
-* `BIOMAGE_EMAIL` is used by the `experiment pull` command to automatically add permissions for the downloaded experiment.
-You should specify the email used to log in into the platform in staging.
+* `BIOMAGE_EMAIL` is used by the `experiment pull/copy` command to automatically add permissions for the downloaded experiment.
+You should specify the email used to log in into the platform in staging. **Note** your current aws account will need permissions
+for cognito.
```

Comment:
Forgot to add to review, `AWS`, `Cognito`

Reply:
what?

Reply:
I mean the capitalisation is missing